.travis.yml

```diff
@@ -4,4 +4,3 @@ scala:
   - 2.11.7
 jdk:
   - oraclejdk7
-  - openjdk7
```

Comment:
We should have both - changes in either direction are material and should go in commit log etc.